source/code/plugin/ApplicationInsightsUtility.rb

```diff
@@ -108,15 +109,21 @@ def initializeUtility()
       rescue => errorStr
         $log.warn("Exception in AppInsightsUtility: initilizeUtility - error: #{errorStr}")
       end
-    end
+    end   
 
-    def getDockerInfo()
-      dockerInfo = DockerApiClient.dockerInfo
-      if (!dockerInfo.nil? && !dockerInfo.empty?)
-        @@CustomProperties["DockerVersion"] = dockerInfo["Version"]
-        #@@CustomProperties["DockerApiVersion"] = dockerInfo["ApiVersion"]
+    def getContainerRuntimeInfo()
+      containerRuntime = ENV[@@EnvContainerRuntime]
+      if !containerRuntime.nil? && !containerRuntime.empty?
+        @@CustomProperties["ContainerRunTime"] = containerRuntime
```

Comment:
How about containerRunTimeVersion for non docker/moby runtimes ?

Reply:
there is no containerRunTimeVersion  in /pods api call so this has to be surfaced from /nodes KubeAPI call. let me see if there is any way to do this.

Reply:
We have telemetry captured in NodeCoreCapacity metric via DockerVersion which has runtime and version so we are good with that.  This one only used for the debugging purpose and here we can only either runtime name for non-docker environments or docker version for docker envs.